lms/templates/instructor/instructor_dashboard_2/edit_coupon_modal.html

```diff
@@ -3,7 +3,8 @@
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
 %>
-<section id="edit-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-label="${_('Edit Coupon Code')}">
+<section id="edit-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-labelledby="header-edit-coupon-code">
+    <h2 id="header-edit-coupon-code">${_("Edit Coupon Code")}</h2>
```

Comment:
I didn't include "modal" in the heading `id` value because while this header is part of the modal container, it itself isn't representative of a modal. If anything, this may be updated to `heading-edit-coupon-code` or `edit-coupon-code-heading`. Would either of those be preferable for you?